.ci/scripts/gather_test_models.py

```diff
@@ -20,16 +20,16 @@
 CUSTOM_RUNNERS = {
     "linux": {
         # This one runs OOM on smaller runner, the root cause is unclear (T163016365)
-        "w2l": "linux.12xlarge",
-        "ic4": "linux.12xlarge",
-        "resnet50": "linux.12xlarge",
-        "llava": "linux.12xlarge",
-        "llama3_2_vision_encoder": "linux.12xlarge",
-        # "llama3_2_text_decoder": "linux.12xlarge",  # TODO: re-enable test when Huy's change is in / model gets smaller.
```

Comment:
Can you attach the job link to this model since we re-enable it ?

Reply:
Good catch, I think I will comment it out and leave it for latter.  It doesn't OOM but take forever to export (close to 6 hours so far).  I don't have much context, so probably need help from @dvorjackz to figure this one out